core/trino-main/src/test/java/io/trino/server/security/oauth2/TestDynamicCallbackOAuth2Service.java

```diff
@@ -44,11 +44,14 @@
     private static final String CLIENT_SECRET = "secret";
 
     private final TestingHydraIdentityProvider hydraIdP = new TestingHydraIdentityProvider();
-    private final HttpClient httpClient = new JettyHttpClient(new HttpClientConfig()
-            .setTrustStorePath(getResource("cert/localhost.pem").getPath()));
+    private HttpClient httpClient;
     private String hydraUrl;
     private SigningKeyResolver signingKeyResolver;
 
+    public TestDynamicCallbackOAuth2Service()
```

Comment:
remove

Reply:
This is required because `private final TestingHydraIdentityProvider hydraIdP = new TestingHydraIdentityProvider();` can throw `IOException`, because it creates a temp dir.

Reply:
since `TestingHydraIdentityProvider` intent usage is as above, we could simply rewrap checked into unckeched in TestingHydraIdentityProvider ctor 

this is tests only, so rewrapping is no harm

Reply:
I was wrong, it was a leftover, removed!